buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -51,28 +51,28 @@ object Repositories {
 
 object Libraries {
     object Versions {
-        const val kotlin = "1.6.21"
-        const val coroutines = "1.6.1-native-mt"
+        const val kotlin = "1.7.10"
+        const val coroutines = "1.6.4"
         const val jetpack = "1.1.0"
         const val constraintLayout = "1.1.3"
-        const val ktx = "1.6.0"
+        const val ktx = "1.8.0"
         const val material = "1.5.0"
         const val pinEditText = "1.2.3"
         const val sqlLiteJdbc = "3.36.0"
         const val desugaring = "1.1.5"
         const val workManager = "2.7.1"
-        const val fragment = "1.2.5"
+        const val fragment = "1.5.1"
         const val compose = "1.2.0-beta01"
```

Comment:
Compose 1.2.1 is in stable 
```suggestion
        const val compose = "1.2.1"
```

Reply:
I raise you `1.3.0`